tools/bin/incrementversion.js

```diff
@@ -36,9 +36,11 @@ function isValidVersion(version) {
     if (version == DEV_VERSION_NAME) {
         return false;
     }
+    /*
     if (!semver.valid(version)) {
```

Comment:
Please add a `validVersion` function here, just with some simpler logic. A test `\d\.x` should probably be fine.